cmd/node/node.go

```diff
@@ -360,8 +366,9 @@ func (app *SpacemeshApp) setupGenesis(state *state.TransactionProcessor, msh *me
 
 	_, err := state.Commit()
 	if err != nil {
-		log.Panic("cannot commit genesis state")
+		app.log.With().Error("cannot commit genesis state", log.Err(err))
```

Comment:
Same comment as above: we usually don't want to print the err _and_ return it, it should be printed once where the error is handled